packages/dotnet/5.0.201/run

```diff
@@ -3,5 +3,23 @@
 # Put instructions to run the runtime
 export DOTNET_CLI_HOME=$PWD
 
-shift
-dotnet bin/Debug/net5.0/$(basename $(realpath .)).dll "$@"
\ No newline at end of file
+case "${PISTON_LANGUAGE}" in
+    basic.net)
+        ;&
+    fsharp.net)
+        ;&
```

Comment:
what are these blanked for?

Reply:
They go to the next case which is the csharp.net one since we want the three of them to do the same thing.